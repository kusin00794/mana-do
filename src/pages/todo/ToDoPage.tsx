import React from 'react';
import { RouteComponentProps } from 'react-router-dom';

import ToDoForm from '../../modules/todo/ToDoForm';

const ToDoPage = (props: RouteComponentProps) => {
  return <ToDoForm {...props} />;
};

export default ToDoPage;
