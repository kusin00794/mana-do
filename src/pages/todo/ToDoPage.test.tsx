import { testSnapshots } from '../../utils/test';
import ToDoPage from './ToDoPage';

describe('<ToDoPage />', () => {
  testSnapshots(ToDoPage, [
    {
      props: {},
      description: 'default render',
    },
  ]);
});
