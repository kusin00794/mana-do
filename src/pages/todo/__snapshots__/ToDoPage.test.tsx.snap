// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ToDoPage /> snapshots default render 1`] = `
<div
  className="ToDo__container row"
>
  <div
    className="Todo__creation col-12"
  >
    <input
      className="Todo__input"
      onKeyDown={[Function]}
      placeholder="What need to be done?"
    />
  </div>
  <div
    className="ToDo__list col-12"
  >
    <div
      className="text-center"
    >
      No to do item with the following status: 
      <strong>
        all
      </strong>
    </div>
  </div>
  <div
    className="Todo__toolbar col-12"
  >
    <div
      className="Todo__tabs"
    >
      <button
        className="Action__btn all disabled"
        onClick={[Function]}
      >
        All
      </button>
      <button
        className="Action__btn active"
        onClick={[Function]}
      >
        Active
      </button>
      <button
        className="Action__btn completed"
        onClick={[Function]}
      >
        Completed
      </button>
    </div>
    <button
      className="Action__btn clear-all"
      onClick={[Function]}
    >
      Clear all todos
    </button>
  </div>
</div>
`;
